src/components/basicComponents/BaseButton.vue

```diff
@@ -22,5 +22,11 @@ defineProps({
     type: String,
     default: "/feedbacks",
   },
+  text: { type: String, required: true },
+  value: {
+    type: String,
+    required: false,
```

Comment:
you don't need to put required if it's set to false

Reply:
and I can see that you have the same default with mode :)

Reply:
done ;)